### PR TITLE
enable use of transform_graph tool with contrib/rnn

### PR DESCRIPTION
**Pull request by pks**
This is a simple fix for using contrib/rnn with the transform_graph (and possibly other tools).

In particular, it fixes the following error:
```
2017-08-24 16:07:41.015752: E tensorflow/tools/graph_transforms/transform_graph.cc:210] Op type not registered 'BlockLSTM' in binary running on ip-172-31-30-181. Make sure the Op and Kernel are registered in the binary running in this process.
```

It's also possibly a remedy for issue #11847.

### CODE REVIEW

**Comment by jhseu** [comment]
Please address the CLA before we look at the change. Thanks!

**Comment by dkurt** [comment]
@pks, unfortunately, it doesn't solve a `ValueError: No op named BlockLSTM in defined operations.` problem mentioned in the issue. But `from tensorflow.contrib.rnn import *` does. Thank you!

**Comment by pks** [comment]
@dkurt the proposed fix here fixes the transform_graph tool, not freeze_graph.py -- but glad that the import works :-)

**Comment by pks** [comment]
I signed it!

File: tensorflow/contrib/rnn/BUILD [hunk 1]
```diff
@@ -346,6 +348,8 @@ tf_kernel_library(
     srcs = [
         "kernels/blas_gemm.cc",
         "kernels/blas_gemm.h",
+        "kernels/lstm_ops.h",
```
Comment (jhseu) [inline tensorflow/contrib/rnn/BUILD]: You shouldn't add these here. Instead add the dependencies
`"gru_ops_op_lib"` and `"lstm_ops_op_lib"` to the target

**Comment by pks** [comment]
That works as well. Thanks!

**Comment by drpngx** [comment]
Jenkins, test this please.

@petewarden any opinion on this PR?

**Comment by sb2nov** [comment]
@petewarden ping?

**Review by petewarden — APPROVED** [review]
Sorry for the slow response, this looks good to me.

**Comment by martinwicke** [comment]
MacOS failure is unrelated.

**Pull request merged**